app/models/author.rb

```diff
@@ -0,0 +1,10 @@
+class Author < ActiveRecord::Base
+  has_many :books
+
+  validates :name, presence: true
+  validates :biography, presence: true
+
+  def five_books
```

Comment:
Don't `List` and `Author` have the same `five_books` method now? I wonder if that's worthing DRYing a little bit.